www_src/pages/project/project.jsx

```diff
@@ -208,11 +213,33 @@ var Project = React.createClass({
 
     });
   },
-  selectPage: function (el) {
-    this.setState({
-      camera: this.cartesian.getFocusTransform(el.coords, this.state.zoom),
-      selectedEl: el.id
-    });
+  /**
+   * Highlight a page in the UI and move camera to center it
+   * @param  {Number|String} id ID of page
+   * @param  {Number|String} type Type of highlight ("selected", "source")
+   */
+  highlightPage: function (id, type) {
```

Comment:
Yeah, I tried it as 2 methods at first, which was more single responsibility-ish, but way less DRY...so...yeah. :cactus: